src/Dependencies/Collections/Internal/HashHelpers.cs

```diff
@@ -34,14 +35,12 @@ internal static class HashHelpers
         // h1(key) + i*h2(key), 0 <= i < size.  h2 and the size must be relatively prime.
         // We prefer the low computation costs of higher prime numbers over the increased
         // memory allocation of a fixed prime number i.e. when right sizing a HashSet.
-        private static readonly int[] s_primes =
-        {
+        private static readonly ImmutableArray<int> s_primes = ImmutableArray.Create(
```

Comment:
❗ This is an unwanted deviation from the original source, and makes it more difficult to adopt changes from dotnet/runtime. See the header at the top of the file for more information.

Reply:
i saw that.  but i see no way that this change would genuinely make adopting any changes from that code difficult.